inst/templates/gitlab-ci.yml

```diff
@@ -0,0 +1,46 @@
+image: rocker/tidyverse
+
+stages:
+  - build
+  - test
+  - deploy
+
+building:
+  stage: build
+  script:
+    - R -e "remotes::install_deps(dependencies = T)"
+    - R -e 'devtools::check()'
+
+# To have the converage percentge appear as a gitlab badge follow these instructions
+# https://docs.gitlab.com/ee/user/project/pipelines/settings.html#test-coverage-parsing
+# The coverage parsing string is
+# Code coverage: \d+\.\d+
+
+testing:
+    stage: test
+    allow_failure: true
+    when: on_success
+    only:
+        - master
+    script:
+        - Rscript -e 'install.pacakges("DT")'
+        - Rscript -e 'cc <- covr::package_coverage; covr::gitlab(coverage = cc); cat("Code coverage:", paste0(format(covr::percent_coverage(cc), nsmall = 1),"%"))'
```

Comment:
This won't work at all as-as, you are not actually calling `package_coverage`, just assigning the function to a different name.

I think this could be simplified to just

```r
covr::gitlab(quiet = FALSE)
```

With the mentioned regex above if you want the automatic coverage badge.